app/controllers/second_factor_controller.rb

```diff
@@ -1,51 +0,0 @@
-class SecondFactorController < ApplicationController
```

Comment:
@featheredtoast I moved this under `UsersController` as I didn't think we need another controller to handle this. I didn't find any major problem with the methods in this controller. The only thing that I though would be useful to highlight is that this controller didn't have any access control checks. That means any user even anon could execute the methods in this controller.